src/main/java/com/yurjinia/common/security/jwt/repository/BlacklistRepository.java

```diff
@@ -0,0 +1,25 @@
+package com.yurjinia.common.security.jwt.repository;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.redis.core.RedisTemplate;
+import org.springframework.stereotype.Service;
+
+import java.util.concurrent.TimeUnit;
+
+@Service
+@RequiredArgsConstructor
+public class BlacklistRepository {
```

Comment:
if you have an annotation 'Service' then the class must have a name Service as well, I would say this really can be moved to CacheService within separate package `common.cache`.

But general idea is awesome - kudos for your architecture knowledge skills! __x__